### DISCUSSION TITLE
Sprint 2 Goal

### DISCUSSION THREAD

**Discussion by Nathan-Turtleduck** [cs481-ekh/f22-kitty-cat-studios#143] [Sprint Planning]
General Team Sprint Goal: new runner model mockup, make new class for runner stats (Runner Configuration), finish new map, adjust powerup behavior

**Comment by timblegoorn** [comment]
AI Team Sprint Goal: Enhance AI Perception (Work on shot angle/view radius, etc) & Spawning System (add randomized spawning points, map max AI)
